### DISCUSSION TITLE
UI: Options: AppGroups: Deprecate Text Edit Fields

### DISCUSSION THREAD

**Discussion by Mobeeuz** [tnodir/fort#210] [Q&A] [answered]
Ack! Why? This made making and adjusting a base ruleset simple while keeping all newly added programs separated across multiple desktops. Creating rulesets one item at a time manually is unnecessarily cumbersome and feature regressive to what was already working...

**Comment by tnodir** [comment, marked as answer]
As written in [v3.11.0](https://github.com/tnodir/fort/releases/tag/v3.11.0):

> Attention: Please move Allow/Block Texts from App Groups to Wildcard Programs!
> (They're read-only now and will be removed in v4.)

You can create a Program entry with Wildcard paths with specific App Group in the Programs window: `Add Wildcard` menu item for each Allow or Block text from the App Groups. They work same way as in App Groups.

I'm going to radically simplify the App Groups:
- extract App Groups from Options to separate sub-window of Programs window
- extract Speed Limits from App Groups to separate window (so Speed Limit will be specified to Program, not App Group)

**Comment by tnodir** [comment]
I used the text fields in App Groups from start of the Fort implementation:

![fort-v2 5](https://github.com/tnodir/fort/assets/303655/1ab64004-16d1-478b-9357-319a280706e1)

Later I've added Programs window, which overrides individual app rules from the App Groups.

Recently I've added Wildcard paths to Programs window.

So I've moved my settings from App Groups to Wildcards.

Example of `Dev` group:

![fort-wild](https://github.com/tnodir/fort/assets/303655/2b6caf93-c332-4ad6-99f4-e5120c221198)

Example of `IExplore` group:

![fort-wild-ie](https://github.com/tnodir/fort/assets/303655/68a71488-8888-40d1-ae98-f96a5b11d099)

**Comment by Mobeeuz** [comment]
I appreciate the information - it works as you specified. Considering the radical change it is for existing users, a mention in the release notes would be helpful.
